utils/testutils/testutils.go

```diff
@@ -31,44 +26,3 @@ func ReadTestFile(name string) ([]byte, error) {
 	}
 	return data.Bytes(), nil
 }
-
```

Comment:
Please revert this. You cannot see this yet, but this function is called from EE code. :)

Reply:
fixed